src/pages/documentation/release-notes/index.md

```diff
@@ -10,6 +10,15 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 * `target.previewEnabled` configuration setting is no longer required to enable preview mode in Target iOS SDK. If not configured, the default value here will be set to true.
 
+### AEP Flutter Plugins (Core, Assurance, Consent, Edge, Edge Identity - 2.0)
+
+* Updated all our AEP Flutter plugins to use Android 2.0
```

Comment:
Mobile SDK for Android 2.0

Reply:
Also, I would suggest renaming this section to something simple (since the title can also be used as anchor for direct links) and listing the wrappers in the description. E.g.:

### AEP Flutter plugins 2.0.0

Updated all the AEP Flutter plugins to use Mobile Core 2.x and compatible extensions.

The following plugins have been published:
- [flutter_aepcore@2.0.0](https://pub.dev/packages/flutter_aepcore)
- [flutter_aepedge@2.0.0](https://pub.dev/packages/flutter_aepedge)
- [flutter_aepedgeidentity@2.0.0](https://pub.dev/packages/flutter_aepedgeidentity)
- [flutter_aepedgeconsent@2.0.0](https://pub.dev/packages/flutter_aepedgeconsent)
- ...

Here is another example https://developer.adobe.com/client-sdks/documentation/release-notes/#aep-react-native-plugins